siteComponents/sidebar.css

```diff
@@ -0,0 +1,53 @@
+/* sidebar Starts */
+.sidebar {
+    width: 100%;
+    height: 100%;
+    background-color: var(--light-theme);
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar__fliter {
+    display: flex;
+    margin-bottom: 2.4rem;
+}
+
+.sidebar__fliter button {
+    margin-left: auto;
```

Comment:
avoid using tag to apply css

Reply:
> .sidebar__fliter button {

tag is used for a specific element inside the sidebar.